tests/k6-test/k6-test-script.js

```diff
@@ -1,26 +1,26 @@
 import http from 'k6/http';
 import { sleep } from 'k6';
 
-export const options = {
+export let options = {
   stages: [
-    { duration: '10s', target: 100 }, // below normal load
-    { duration: '1m', target: 100 },
-    { duration: '10s', target: 1400 }, // spike to 1400 users
-    { duration: '3m', target: 1400 }, // stay at 1400 for 3 minutes
-    { duration: '10s', target: 100 }, // scale down. Recovery stage.
-    { duration: '3m', target: 100 },
-    { duration: '10s', target: 0 },
+    { duration: "10s", target: 100 }, // below normal load
+    { duration: "1m", target: 100 },
+    { duration: "10s", target: 1400 }, // spike to 1400 users
+    { duration: "3m", target: 1400 }, // stay at 1400 for 3 minutes
+    { duration: "10s", target: 100 }, // scale down. Recovery stage.
+    { duration: "3m", target: 100 },
+    { duration: "10s", target: 0 },
   ],
 };
 export default function () {
   const BASE_URL = 'http://localhost:4000'; // make sure this is not production
 
-  const responses = http.batch([
+  let responses = http.batch([
     [
       'GET',
       `${BASE_URL}/shop/${Math.floor(Math.random() * 9000)}/${Math.floor(
-        Math.random() * 7
-      )}`,
+        Math.random() * 6
+      ) + 1}`,
       null,
       null,
     ],
```

Comment:
Can you clarify your logic of why these were changed to const instead of let while leaving line 16 as const?

Reply:
Oops, I was testing if let made a difference. But I'll revert it back to const.